.buildkite/full_pipeline_trigger.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+if [[ "$BUILDKITE_MESSAGE" == *"[QUICK CI]"* ]]; then
+  PIPELINE='pipeline.smoke.yml'
+elif [[ "$BUILDKITE_MESSAGE" == *"[FULL CI]"* ||
```

Comment:
Could we make the commit string lower case, i.e. `[full ci]`?  It would then be consistent with Buidlkite's `[skip ci]` construct and I think a little easier to type.

Reply:
...or do a case-insensitive string comparison, of course.